community/examples/gke-tpu-v4/gke-tpu-v4-deployment.yaml

```diff
@@ -40,7 +40,7 @@ vars:
 
   # The number of nodes to be created in each nodepool
   # For TPUs, static_node_count should be num of chips based on topology divided by num chips for the machine type
-  # Reference: https://cloud.google.com/tpu/docs/v4
+  # Example: https://cloud.google.com/tpu/docs/v4
```

Comment:
nit: This looks like a reference and not an example. Any reason to define this as an example?

Reply:
I agree, will revert it.